src/com/projectkorra/projectkorra/Element.java

```diff
@@ -96,12 +96,31 @@ public Element(String name, ElementType type) {
 	 * @param type ElementType specifies if its a regular element or chi style
 	 *            element.
 	 * @param plugin The plugin that is adding the element.
+	 * @param color Element's color
+	 * @param prefix Element's prefix
 	 */
-	public Element(String name, ElementType type, Plugin plugin) {
+	public Element(String name, ElementType type, Plugin plugin, String color, String prefix) {
 		this.name = name;
 		this.type = type;
 		this.plugin = plugin;
 		ALL_ELEMENTS.put(name.toLowerCase(), this);
+		if(getElement(name) instanceof SubElement) {
+			String mainName = prefix;
+			if(((ConfigManager.languageConfig.get().getString("Chat.Colors." + mainName + "Sub")) == null)) {

```

Comment:
It may also be worth noting that instead of simply changing the constructors that are currently there, you should add new constructors that support the use of the `ChatColor` color and `String` prefix arguments in order to allow it to be backward compatible with peoples current addons.